NanaZipPackage/Package.appxmanifest

```diff
@@ -153,16 +153,15 @@
         </uap:Extension>
         <desktop4:Extension Category="windows.fileExplorerContextMenus">
           <desktop4:FileExplorerContextMenus>
-            <!-- Our verb name starts with "C" to work around a bug preventing our shell menu from appearing in the
-                 classic context menu. -->
+            <!-- Use a low name prefix to work around our shell menu not appearing in the classic context menu. -->
```

Comment:
I hope you can add the issue link https://github.com/MediaArea/MediaInfo/pull/998, which you have provided in this PR's conversation, that can be helpful to know the reason better.